File: dal-dao-gen/dal_ftp/63/java/Entity/Accountcheck.java
package com.ctrip.dal.test.test4;

import java.math.BigDecimal;
import java.sql.Timestamp;

import com.ctrip.platform.dal.dao.DalPojo;

public class Accountcheck implements DalPojo {
	private Integer accCheckID;
	private String corpID;
	private String batchNo;
	private Integer accountID;
	private Integer subAccountID;
	private String batchStatus;
	private Integer accBalanceID;
	private String accountType;
	private Integer checkAccType;
	private String operator;
	private Timestamp modifyTime;
	private String startDate;
	private String endDate;
	private BigDecimal fltconMoney;
	private BigDecimal htlHconMoney;
	private BigDecimal htlXconMoney;
	private BigDecimal limited;
	private BigDecimal limitedTemp;
	private Timestamp datachangeLasttime;
	public Integer getAccCheckID() {
		return accCheckID;
	}

	public void setAccCheckID(Integer accCheckID) {
		this.accCheckID = accCheckID;
	}

	public String getCorpID() {
		return corpID;
	}

	public void setCorpID(String corpID) {
		this.corpID = corpID;
	}

	public String getBatchNo() {
		return batchNo;
	}

	public void setBatchNo(String batchNo) {
		this.batchNo = batchNo;
	}

	public Integer getAccountID() {
		return accountID;
	}

	public void setAccountID(Integer accountID) {
		this.accountID = accountID;
	}

	public Integer getSubAccountID() {
		return subAccountID;
	}

	public void setSubAccountID(Integer subAccountID) {
		this.subAccountID = subAccountID;
	}

	public String getBatchStatus() {
		return batchStatus;
	}

	public void setBatchStatus(String batchStatus) {
		this.batchStatus = batchStatus;
	}

	public Integer getAccBalanceID() {
		return accBalanceID;
	}

	public void setAccBalanceID(Integer accBalanceID) {
		this.accBalanceID = accBalanceID;
	}

	public String getAccountType() {
		return accountType;
	}

	public void setAccountType(String accountType) {
		this.accountType = accountType;
	}

	public Integer getCheckAccType() {
		return checkAccType;
	}

	public void setCheckAccType(Integer checkAccType) {
		this.checkAccType = checkAccType;
	}

	public String getOperator() {
		return operator;
	}

	public void setOperator(String operator) {
		this.operator = operator;
	}

	public Timestamp getModifyTime() {
		return modifyTime;
	}

	public void setModifyTime(Timestamp modifyTime) {
		this.modifyTime = modifyTime;
	}

	public String getStartDate() {
		return startDate;
	}

	public void setStartDate(String startDate) {
		this.startDate = startDate;
	}

	public String getEndDate() {
		return endDate;
	}

	public void setEndDate(String endDate) {
		this.endDate = endDate;
	}

	public BigDecimal getFltconMoney() {
		return fltconMoney;
	}

	public void setFltconMoney(BigDecimal fltconMoney) {
		this.fltconMoney = fltconMoney;
	}

	public BigDecimal getHtlHconMoney() {
		return htlHconMoney;
	}

	public void setHtlHconMoney(BigDecimal htlHconMoney) {
		this.htlHconMoney = htlHconMoney;
	}

	public BigDecimal getHtlXconMoney() {
		return htlXconMoney;
	}

	public void setHtlXconMoney(BigDecimal htlXconMoney) {
		this.htlXconMoney = htlXconMoney;
	}

	public BigDecimal getLimited() {
		return limited;
	}

	public void setLimited(BigDecimal limited) {
		this.limited = limited;
	}

	public BigDecimal getLimitedTemp() {
		return limitedTemp;
	}

	public void setLimitedTemp(BigDecimal limitedTemp) {
		this.limitedTemp = limitedTemp;
	}

	public Timestamp getDatachangeLasttime() {
		return datachangeLasttime;
	}

	public void setDatachangeLasttime(Timestamp datachangeLasttime) {
		this.datachangeLasttime = datachangeLasttime;
	}

}

File: dal-dao-gen/dal_ftp/63/cs/Entity/Accountcheck.cs
using System;
using Arch.Data.Orm;

namespace com.ctrip.dal.test.test4.Entity.DataModel
{
    /// <summary>
    /// _accountcheck
    /// </summary>
    [Serializable]
    [Table(Name = "_accountcheck")]
    public partial class Accountcheck
    {
        /// <summary>
        /// 获取或设置AccCheckID
        /// </summary>
        [Column(Name = "AccCheckID",Length=10),ID]
        public int AccCheckID { get; set; }
        /// <summary>
        /// 获取或设置CorpID
        /// </summary>
        [Column(Name = "CorpID",Length=20)]
        public string CorpID { get; set; }
        /// <summary>
        /// 获取或设置BatchNo
        /// </summary>
        [Column(Name = "BatchNo",Length=60)]
        public string BatchNo { get; set; }
        /// <summary>
        /// 获取或设置AccountID
        /// </summary>
        [Column(Name = "AccountID",Length=10)]
        public int AccountID { get; set; }
        /// <summary>
        /// 获取或设置SubAccountID
        /// </summary>
        [Column(Name = "SubAccountID",Length=10)]
        public int? SubAccountID { get; set; }
        /// <summary>
        /// 获取或设置BatchStatus
        /// </summary>
        [Column(Name = "BatchStatus",Length=1)]
        public char BatchStatus { get; set; }
        /// <summary>
        /// 获取或设置AccBalanceID
        /// </summary>
        [Column(Name = "AccBalanceID",Length=10)]
        public int? AccBalanceID { get; set; }
        /// <summary>
        /// 获取或设置AccountType
        /// </summary>
        [Column(Name = "AccountType",Length=1)]
        public char AccountType { get; set; }
        /// <summary>
        /// 获取或设置CheckAccType
        /// </summary>
        [Column(Name = "CheckAccType",Length=10)]
        public int CheckAccType { get; set; }
        /// <summary>
        /// 获取或设置Operator
        /// </summary>
        [Column(Name = "Operator",Length=20)]
        public string Operator { get; set; }
        /// <summary>
        /// 获取或设置ModifyTime
        /// </summary>
        [Column(Name = "ModifyTime",Length=23)]
        public DateTime ModifyTime { get; set; }
        /// <summary>
        /// 获取或设置StartDate
        /// </summary>
        [Column(Name = "StartDate",Length=8)]
        public char? StartDate { get; set; }
        /// <summary>
        /// 获取或设置EndDate
        /// </summary>
        [Column(Name = "EndDate",Length=8)]
        public char? EndDate { get; set; }
        /// <summary>
        /// 获取或设置FltconMoney
        /// </summary>
        [Column(Name = "FltconMoney",Length=19)]
        public decimal? FltconMoney { get; set; }
        /// <summary>
        /// 获取或设置HtlHconMoney
        /// </summary>
        [Column(Name = "HtlHconMoney",Length=19)]
        public decimal? HtlHconMoney { get; set; }
        /// <summary>
        /// 获取或设置HtlXconMoney
        /// </summary>
        [Column(Name = "HtlXconMoney",Length=19)]
        public decimal? HtlXconMoney { get; set; }
        /// <summary>
        /// 获取或设置limited
        /// </summary>
        [Column(Name = "limited",Length=19)]
        public decimal? Limited { get; set; }
        /// <summary>
        /// 获取或设置LimitedTemp
        /// </summary>
        [Column(Name = "LimitedTemp",Length=19)]
        public decimal? LimitedTemp { get; set; }
        /// <summary>
        /// 获取或设置DataChangeLastTime
        /// </summary>
        [Column(Name = "DataChangeLastTime",Length=23)]
        public DateTime? DataChangeLastTime { get; set; }
    }
}